qase-api/src/main/java/io/qase/api/aspects/QaseConfigAutoReloadAspect.java

```diff
@@ -0,0 +1,58 @@
+package io.qase.api.aspects;
+
+import io.qase.api.config.QaseConfig;
+import org.aeonbits.owner.Reloadable;
+import org.aspectj.lang.annotation.After;
+import org.aspectj.lang.annotation.AfterReturning;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Pointcut;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+@Aspect
```

Comment:
I think reloading by aspect is bad. Aspect will run every time when System.setProperties will be used. This feature needs only for our autotest, the user does not need it.

Reply:
@savkk, thank you for the review. 
The commits have been rolled back. 

It seemed like an interface for end users to manipulate `QASE_*` environment variables, so I decided to avoid the necessity to invoke `Config.reload()` 'manually' each time the system properties are updated.